pkg/insights/insightsreport/insightsreport.go

```diff
@@ -261,11 +265,17 @@ type healthStatusCounts struct {
 	critical, important, moderate, low, total int
 }
 
-func (c *Controller) readInsightsReport(report types.SmartProxyReport) ([]types.InsightsRecommendation, healthStatusCounts, time.Time) {
+func (c *Controller) readInsightsReport(report types.SmartProxyReport) ([]types.InsightsRecommendation, healthStatusCounts, time.Time, error) {
 	healthStatus := healthStatusCounts{}
 	healthStatus.total = report.Meta.Count
 	activeRecommendations := []types.InsightsRecommendation{}
 
+	clusterVersion, err := c.client.GetClusterVersion()
+	if err != nil {
+		klog.Errorf("Unable to extract cluster version: %v", err)
+		return nil, healthStatus, time.Time{}, err
+	}
```

Comment:
I think this call has nothing to do with reading the Insights report :) ....but I am thinking how to do it better...